internal/cmd/connect/command_custom_plugin_create.go

```diff
@@ -0,0 +1,83 @@
+package connect
+
+import (
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/spf13/cobra"
+	"path/filepath"
+)
+
+type pluginCreateOut struct {
+	Id         string `human:"ID" serialized:"id"`
+	Name       string `human:"Name" serialized:"name"`
+	ErrorTrace string `human:"Error Trace,omitempty" serialized:"error_trace,omitempty"`
+}
+
+func (c *customPluginCommand) newCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "create <name>",
+		Short:       "Create a custom connector plugin.",
+		Args:        cobra.ExactArgs(1),
+		RunE:        c.createCustomPlugin,
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Create a custom connector plugin ",
+				Code: "confluent connect custom-plugin create plugin_name --config-file config.json",
```

Comment:
Please take a look at other commands for examples of how to format the example field.
```suggestion
				Text: `Create custom connector plugin "my-plugin".`,
				Code: "confluent connect custom-plugin create my-plugin --config-file config.json",
```

Reply:
I had used this example for formatting https://github.com/confluentinc/cli/blob/main/internal/connect/command_cluster_create.go#L28

Is there a different example I should base my formatting of?